modules/models/scheduled_message_models.py

```diff
@@ -0,0 +1,52 @@
+from datetime import datetime
+from pydantic import Field
+from enum import Enum
+
+from modules.models.shared_models import AirtableRowBaseModel
+
+
+class FrequencyEnum(str, Enum):
+    daily = "daily"
+    weekly = "weekly"
+    monthly = "monthly"
+
+
+class ScheduledMessageInfo(AirtableRowBaseModel):
```

Comment:
Does this do something that the built in `/remind` doesn't do? I'm fine either way, just curious if there's a specific benefit

Reply:
Remind has to be refreshed, this would be ran by the background job/process and would only need updates on Airtable.